ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__arm64-stable-openshift-logging-6.3.yaml

```diff
@@ -47,7 +47,7 @@ resources:
       memory: 1Gi
 tests:
 - as: aws-ipi-fips-arm-f999-logging
-  cron: 5 11 29 2 *
+  cron: 39 3 13,18 * *
```

Comment:
This is a frequency increase.  Maybe we can skip the changes for repo `openshift/openshift-tests-private` ?

/cc @jianlinliu @anpingli